### ISSUE TITLE
VSCodium support & availability on open-vsx

### ISSUE THREAD

**Issue by stashymane** [godotengine/godot-csharp-vscode#12]
VSCodium is a fork of the open source part of VSCode, mainly focusing on removing telemetry & proprietary code, which in turn prevents it from using Microsoft's extension marketplace. Is it possbile to make this extension compatible with vscodium and available on open-vsx (its extension marketplace)?

**Comment by neikeq** [comment]
We only depend on `ms-vscode.mono-debug` because we re-use some of their settings, so with some changes the dependency could be removed.

**Comment by geekley** [comment]
Yes, please remove the dependency, so we can at least install the .vsix file manually for now.

**Comment by copygirl** [comment]
I have the `muhammad-sammy.csharp` and `ms-vscode.mono-debug` (installed via .vsix) extensions installed. I downloaded the .vsix for this extension, edited the `package.json` and removed the dependency on `ms-dotnettools.csharp`, and installed it. This appears to work for me.

(I would however appreciate a release on open-vsx.)

**Comment by raulsntos** [comment]
Seems like this would be resolved by https://github.com/open-vsx/publish-extensions/pull/565

**Comment by manuth** [comment]
> Seems like this would be resolved by [open-vsx/publish-extensions#565](https://github.com/open-vsx/publish-extensions/pull/565)

I'm affraid I have to do a little a fixup as I forgot to edit this extension's metadata.

Although, having this extension published to open-vsx through the open-vsx/publish-extensions should be considered a workaround.

Thanks a ton for this great work 😄

**Comment by manuth** [comment]
@stashymane, @copygirl, @raulsnots
Just for you guys to know - the `neikeq.godot-csharp-vscode` is published to open-vsx automatically as of now!
However, I'd recommend to leave this issue open as it's recommended to publish extensions using the `ovsx` CLI (see https://www.npmjs.com/package/ovsx) rather than using the `open-vsx/publish-extensions` procedure, as that project is actually meant to be used for publishing abandoned extensions.

Thanks for your patience!

@neikeq in order to be able to publish `neikeq.godot-csharp-vscode` to OpenVSX, I dropped both the `ms-vscode.mono-debug`- and the `ms-dotnettools.csharp`-dependencies and instead added `muhammad-sammy.csharp` as a dependency.

I tried to run a project, set some debug break points and attach to a running editor. So I consider the extension I published to OpenVSX as a valid workaround for now.
